frontend/packages/operator-lifecycle-manager/src/components/operand.tsx

```diff
@@ -64,12 +64,19 @@ const csvName = () =>
     );
 
 const getActions = (selectedObj: any) => {
```

Comment:
Not new, but we should have a type other than `any` here. It's hard to recommend a fix since it's unclear what kind of obj this is.